README.md

```diff
@@ -7,11 +7,39 @@ Build tools for the Seam API using this blueprint.
 
 ## Description
 
-TODO
+A blueprint is a simplified description of an API schema that is optimized for code and documentation generation.
+
+The blueprint schema assumes the API follows Seam API design guidelines,
+and will evolve with those guidelines and any SDK, documentation, or product quality requirements.
+
+The blueprint defines the following core concepts:
+
+- Endpoint: a path that accepts an HTTP request and returns a response.
+- Route: the direct parent path of a collection of one or more endpoints.
+- Subroute: A route that is nested under another route.
+- Namespace: the direct parent path of a collection routes only: there are no endpoints under this path.
+- Resource: every endpoint that returns non-empty data returns a resource or collection of resources.
 
 ## Motivation
 
-TODO
+The scope of the [OpenAPI Specification](https://swagger.io/specification/) covers arbitrarily complicated APIs,
+while the Seam API follows very predictable and simplified API design patterns.
```

Comment:
Is the line break intentional here? 
```suggestion
The scope of the [OpenAPI Specification](https://swagger.io/specification/) covers arbitrarily complicated APIs, while the Seam API follows very predictable and simplified API design patterns.
```

Reply:
I break long lines in markdown intentionally. It doesn't change the format and it makes diffs / comments easier to target.

Note that linebreaks in github comments / issues do change the format (no idea why their parser is different 🤷🏻 ).